Yoast/wordpress-seo
# Release of package Yoastseo 1.91.1

## Context


* Release NPM package yoastseo version 1.91.1 to provide Vite support.

## Summary


This PR can be summarized in the following changelog entry:

* Upgrades NPM package yoastseo to version 1.91.1 to add Vite support.

## Relevant technical choices:

* The fix from #19361 has already been merged into `trunk`, this only contains the changelog update of 1.91.1 and the package.json version update.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

**Test within the plugin**

* Build the plugin as usual
* We have 3 packages that have `yoastseo` as a dependency, so test their usages:
  * `search-metadata-previews`: Visit a post and check in the metabox & sidebar that the search preview still work. Specifically the highlighting (desktop view focus keyphrase should be bold) and length progress bars should work as before.
  * `replacement-variable-editor`: Visit a post and check in the metabox & sidebar, as well as the settings, and check that our replacement variables still work -- note that `yoastseo` is not actually used, so please don't spend too much time here.
  * `yoast-components`: Visit a post and check in the metabox & sidebar that the keyphrase inputs still work. Specifically the RTL styles of the label and remove button (only Premium related keyphrases)

**Test the published version**

* Create your own local Vite project following the steps at https://vitejs.dev/guide/#scaffolding-your-first-vite-project. Please use the React base.
* Run `yarn add yoastseo@1.91.1 lodash util`
* In your local Vite project, edit `src/App.jsx`:
  * At the end of the import statements, add `import {string} from 'yoastseo';`
  * After the rendered h1 from the App component, add a newline containing: `<pre>{ string.stripSpaces( '   Hello      world!   ' ) }</pre>`
* In your local Vite project, edit `vite.config.js`:
  * Under plugins add: `define: { 'process.env': process.env }`
* Run `$ yarn dev`
* The Vite app should show "Hello world!" below the header, without excessive whitespaces. 

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* We have 3 packages that have `yoastseo` as a dependency, so those could theoretically be affected: `search-metadata-previews`, `replacement-variable-editor`, `yoast-components`.

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes #
